dbeam-core/src/main/java/com/spotify/dbeam/avro/BeamJdbcAvroSchema.java

```diff
@@ -55,8 +55,7 @@ public static Schema createSchema(Pipeline pipeline, JdbcExportArgs args, Connec
     final long elapsedTimeSchema = (System.nanoTime() - startTime) / 1000000;
     LOGGER.info("Elapsed time to schema {} seconds", elapsedTimeSchema / 1000.0);
 
-    JobNameConfiguration.configureJobName(
-        pipeline.getOptions(), dbName, args.queryBuilderArgs().tableName());
+    JobNameConfiguration.configureJobName(pipeline.getOptions(), dbName);
```

Comment:
Can we keep the table name (when available) as part of the `jobName`? Often the exported table is a 1-1 mapping to the output endpoint. I guess we can take `tableName` from `args`..

Reply:
Done.